solvexia_sdk/file/file.py

```diff
@@ -0,0 +1,77 @@
+#!/usr/bin/env python
+
+import requests
+import json
+import sys
+import math
+import os
+from fsplit.filesplit import Filesplit
+from solvexia_sdk import api
+
+class file:
+    def __init__(self, fileId):
+        self.fileId = fileId
+
+    def getFileMetadata(self):
+        response = api.apiGet(f"files/{self.fileId}/metadata")
+        api.statusCodeCheck(response, "Error getting metadata for file")
+        return response.json()
+
+    def updateMetadata(self, name):
+        payload = {
+            'name': name
+        }
+        response = api.apiPost(f"files/{self.fileId}/metadata", payload)
+        api.statusCodeCheck(response, "Error updating file metadata")
+        return response.json()
+
+    def uploadFile(self, file):
+        uploadFileUrl = api.baseUrl + f"files/{self.fileId}"
+        with open(file, "rb") as openFile:
+            response = requests.post(uploadFileUrl, files={"Filename": openFile}, headers=api.access_token)
+        api.statusCodeCheck(response, "Error uploading file")
+        print(response.json())
+        return response.json()
+
+    def downloadFile(self):
+        downloadFileUrl = api.baseUrl + f"files/{self.fileId}"
+        headers = api.access_token
```

Comment:
`api.access_token` is json object and assigning it to headers, would make headers an object. But, then on following line (line#39) headers is accessed and assigned as if it is an array.

Is this allowed in python?

Reply:
A json object in python has a similar structure to a dictionary where there are keys and corresponding values. We can then add additional elements to the json object by specifying a new key and value, similar to the way it is done in an array.